mqtt/src/test/scala/akka/stream/alpakka/mqtt/scaladsl/MqttSourceSpec.scala

```diff
@@ -41,6 +41,7 @@ class MqttSourceSpec
   val topic1 = "source-spec/topic1"
   val topic2 = "source-spec/topic2"
   val secureTopic = "source-spec/secure-topic1"
+  val willTopic = "source-spec/will"
```

Comment:
Add this to `src/test/travis/acl` under the anonymous section. Otherwise no one can publish to this topic.